### PR TITLE
JavaScript capstone project: Your API-based webapp

### PR DESCRIPTION
**Pull request by Ibtisam34**
**The JavaScript capstone project is about building our own web application based on an external API. we had selected movie API. The web app has 2 user interfaces .** 

- [ ] A  home page showing a list of items that you can "like."

- [ ]  A popup window with more data about an item that you can use to comment on.

** the PR is by @Ibtisam34  and @jodavid444 **

**our kanban-board https://github.com/Ibtisam34/kanban-board/projects/1**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+};
+
+const postNewComments = (movieID, userName, userComment) => {
+  axios.post('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments', {
+    item_id: movieID,
+    username: userName,
+    comment: userComment,
+  });
+  showComments(movieID);
+};
```
Comment (codepantha) [inline src/index.js]: - [ ] [OPTIONAL] It is expected according to the project requirements that when you submit a new comment, the data gets POSTED to the involvement API and the screen gets updated. Please see below 👇 
image

**Review by codepantha — COMMENTED** [review]
# STATUS: OPTIONAL CHANGES REQUIRED ♻️ 

## Hi team 👋, 

Good job so far! It is obvious that you have put in so much effort into this project 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used GitFlow correctly
✔️ No linter errors
✔️ Good commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ibtisam34** [comment]
- [ ]  hello sir @codepantha from myside the involvement is working plz can we meet on call thanxs

**Comment by Ibtisam34** [comment]
- [ ]  hello sir @codepantha  my dashboard is still pending plz make changes I cant resubmit the pull request

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
```
Comment (ShadyShawkat) [inline src/index.js]: - [ ] [OPTIONAL] As noted by the previous reviewer, please make sure that when the user submits a comment, the comment gets posted to the API and the UI gets updated with the new comment. Currently, I have to refresh the page to see the new comment.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+      <div id='comment-feature'>
+        <p> Rating: ${movies[parseInt(target, 10) - 1].rating.average}</p>
+        <p> Genres: ${movies[parseInt(target, 10) - 1].genres}</p>
+        <p> Released Date: ${movies[parseInt(target, 10) - 1].premiered}</p>
+        <p> Language: ${movies[parseInt(target, 10) - 1].language}</p>
+      </div>
+    </div>`;
+};
+
+// eslint disable
```
Comment (ShadyShawkat) [inline src/index.js]: - [ ] [OPTIONAL] Kindly refrain from using `eslint disable` anywhere in your code, as that defies the purpose of using linters in the first place.

File: README.md [hunk 4]
```diff
@@ -1 +1,173 @@
...
+- GitHub: [@jodavid444](https://github.com/jodavid444)
+- Twitter: [@jodavid444dave](https://twitter.com/jodavid444dave)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/joseph-david-01a8a5243)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
```
Comment (ShadyShawkat) [inline README.md]: - [ ] [OPTIONAL] Let's add some of the future features you'd like to implement in the future.

**Review by ShadyShawkat — COMMENTED** [review]
Hi Team, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Ibtisam34** [comment]
Plz help me this part Iam remaining  only 1 review  and I already repeated
the week

**Comment by codepantha** [comment]
> * [ ]   hello sir @codepantha from myside the involvement is working plz can we meet on call thanxs

I apologize for the late response. All changes are optional. The problem is that you have to reload the page for the new comment to show. The correct behaviour would be for the comments to show up automatically after interacting with the involvement API.

**Comment by codepantha** [comment]
> * [ ]   hello sir @codepantha from myside the involvement is working plz can we meet on call thanxs

I apologize for the late response. All changes for this milestone are optional. Yes, it's working but the problem is that you have to reload the page for the new comment to show. The correct behaviour would be for the comments to show up automatically after interacting with the involvement API.

**Pull request merged**